Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/EdgeIdentityTab.java

```diff
@@ -136,4 +222,85 @@ public void run() {
             }
         });
     }
-}
\ No newline at end of file
+
+    /**
+     * Async method that retrieves the ad ID from the {@link AdvertisingIdClient} (using Google's gms.ads SDK).
+     * Sanitizes ad ID value to the empty string ({@code ""}) if ad tracking is disabled, or an exception is encountered.
+     *
+     * @param callback receives the ad ID {@code String} if a valid value can be retrieved, {@code ""} otherwise.
+     */
+    private void getAdvertisingIdentifier(final AdobeCallback<String> callback) {
```

Comment:
Is this method called from anywhere? It looks like the code above calls `getAdvertisingIdClientInfo` directly. Can you remove this if it is not used?